robot_simulation.py

```diff
@@ -0,0 +1,58 @@
+import matplotlib.pyplot as plt
+from dataclasses import dataclass
+import numpy as np
+import click
+
+@dataclass
+class Robot:
+    x: float = 0
+    y: float = 0
+    yaw: float = 0
+
+    v_x: float = 0
+    v_y: float = 0
+    v_yaw: float = 0
+
+    def update(self, dt):
+        self.x += dt * (self.v_x * np.cos(self.yaw))
+        self.y += dt * (self.v_x * np.sin(self.yaw))
+    
+    def show(self):
+        print(f"x:{self.x} y:{self.y}")
+
+    def rotate(self):

```

Comment:
Can you add the rotation direction here please?
```suggestion
    def rotate(self, v_yaw):
```